examples/exceptions/exception.php

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env php
+<?php
+use Amp\Deferred;
+use Amp\Loop;
+
+require_once __DIR__ . "/../../vendor/autoload.php";
+
+function asyncOperation() {
+    $def = new Deferred();
+
+    \Amp\Loop::delay(1000, function () use ($def) {
+        $def->fail(new Exception("force exception")); //this will be catch-ed
+
+//        throw new Exception("force exception"); //this will be NOT catch-ed
```

Comment:
I think there should be a second async function that throws inside the `delay` callback and a second `Loop::run`.

Reply:
@kelunik OK will add 2 example catch-ed & uncatch-ed and PR